src/App.js

```diff
@@ -27,6 +28,7 @@ function App() {
         <Route path="/watchlater" element={<Watchlater />} />
         <Route path="/liked" element={<Liked />} />
         <Route path="/playlist" element={<Playlist />} />
+        <Route path="/video/:videoId" element={<VideoPage />} />
```

Comment:
Appreciation :-good to see u r using dynamic routes for VideoPage products/Items